ci/test-local-cluster.sh

```diff
@@ -0,0 +1 @@
+test-stable.sh
```

Comment:
I don't think this is going to have the desired affect.  If you want test-stable.sh to drop in to the test-local-cluster switch statement, then test-local-cluster.sh needs to be a symlink pointing at test-stable.sh.

Reply:
hehe it is a symlink, I think it's just the way git handles symlinks, for example, test-stable-perf does the same thing: https://github.com/solana-labs/solana/blob/master/ci/test-stable-perf.sh

Reply:
Huh, okay.  In the link you sent, the file actually says the word "Symbolic link" but maybe that's cause it's not showing in diff view, which is still weird it doesn't display it here.  Whatever, I trust you if you say so. :)